service/src/main/java/uk/gov/hmcts/reform/fnp/exception/PaymentsApiException.java

```diff
@@ -8,4 +8,8 @@ public PaymentsApiException(int status, String message, Throwable cause) {
         super(message, cause);
         this.status = status;
     }
+
+    public int getStatus() {
+        return status;
+    }
```

Comment:
Is this status actually ever used anywhere? I think it is the same for the fee register exception. Is this actually useful to store? If so it could be replaced with an `@Getter` if you want to reduce boilerplate.

Reply:
done